src/field/pokemon.ts

```diff
@@ -3746,6 +3746,18 @@ export class EnemyPokemon extends Pokemon {
       this.status = new Status(Overrides.OPP_STATUS_OVERRIDE);
     }
 
+    if (Overrides.OPP_GENDER_OVERRIDE) {
+      this.gender = Overrides.OPP_GENDER_OVERRIDE;
+    }
+
+    const speciesId = this.species.speciesId;
+
+    if (speciesId in Overrides.OPP_FORM_OVERRIDES &&
+      Overrides.OPP_FORM_OVERRIDES[speciesId] &&
+      this.species.forms[Overrides.OPP_FORM_OVERRIDES[speciesId]]) {
```

Comment:
```suggestion
    if (speciesId in Overrides.OPP_FORM_OVERRIDES
      && Overrides.OPP_FORM_OVERRIDES[speciesId]
      && this.species.forms[Overrides.OPP_FORM_OVERRIDES[speciesId]]) {
```
I think this makes it a bit easier to read here.